ledger/src/blockstore_processor.rs

```diff
@@ -70,7 +70,7 @@ fn execute_batch(
         balances,
     ) = batch.bank().load_execute_and_commit_transactions(
         batch,
-        MAX_RECENT_BLOCKHASHES,
```

Comment:
my quick git time-travel told that this bug is introduced sooo long ago: #1499 201**8** Oct PR.